backend/models/risk.js

```diff
@@ -179,7 +179,7 @@ class Risk extends Ticket {
 					oldRisk._id,
 					oldRisk.comments,
 					data,
-					createViewPoint(residualData.viewpoint)
+					createViewPoint(oldRisk._id, residualData.viewpoint)
```

Comment:
@Charence  can you investigate what this is for?

This is the only place that is calling L145, I'm not sure why a mitigation comment is saving a viewpoint (surely that's a property change kind of update?)

Above all the function parameter doesn't even match, so I highly doubt this is used anywhere

Reply:
@carmenfan Risks also has `addMitigationComment`, which I think should all live within `Comment`

Reply:
@Charence 
I'm less worried about where the function is but whether does addMitigationComment ever comes in with a viewpoint.

because as it stands it never triggers as it's passing the wrong function parameters.

If it never has a viewpoint then we should just remove the createViewpoint function

Reply:
@carmenfan Actually, there are a few places that make reference to a `residual` field, but none currently exists so this bit of code doesn't do anything. I'm not sure why we wanted to treat `residual` differently in the first place.